src/DrawerLayout.js

```diff
@@ -212,7 +212,7 @@ export default class DrawerLayout extends Component {
   }
 
   _shouldSetPanResponder = (e, { moveX, dx, dy }) => {
-    if (!dx || !dy) {
+    if (!dx || !dy || Math.abs(dx) < 3) {
```

Comment:
If I'm correct, this value will depend on the device's pixel density. It might still be an issue with devices with higher pixel density. At least that's what I have seen. Probably needs a bit more testing.